### PR TITLE
fixes for noetic/python3 compile/operation

### PR DESCRIPTION
**Pull request by davefeilseifer**
updated to build/run in noetic with python3 passes prerelease test:

mkdir -p /tmp/prerelease_job
cd /tmp/prerelease_job
generate_prerelease_script.py \
  https://raw.githubusercontent.com/ros-infrastructure/ros_buildfarm_config/production/index.yaml \
  noetic default ubuntu focal amd64 \
  --custom-repo \
    pr2_power_drivers__custom-2:git:https://github.com/PR2/pr2_power_drivers.git:kinetic-devel \
    rqt_pr2_dashboard__custom-3:git:https://github.com/PR2/rqt_pr2_dashboard.git:kinetic-devel \
  --level 0 \
  --output-dir ./

### CODE REVIEW

File: scripts/rqt_pr2_dashboard [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```
Comment (v4hn) [inline scripts/rqt_pr2_dashboard]: Same as in https://github.com/PR2/linux_networking/pull/5#discussion_r584112906 .
Please make sure the file is correctly installed via `catkin_install_python` instead and test whether it still works.

Comment (davefeilseifer) [reply scripts/rqt_pr2_dashboard]: fixed

**Comment by davefeilseifer** [comment]
ready for re-review and merge @v4hn

**Comment by davefeilseifer** [comment]
ready for re-review and merge @v4hn

**Review by v4hn — APPROVED** [review]

**Comment by v4hn** [comment]
sorry, I missed the first reminder last week.

**Pull request merged**